wayland/getting-started.txt

```diff
@@ -0,0 +1,292 @@
+    
+[1.0] Using Xwayland
+________________________________________________________________________________
+
+In general, these steps can be followed on a system already running with Xorg.
+Only a few package rebuilds are required to enable Wayland support.
+
+Mesa must be rebuilt first, adding wayland to the list of platforms. 
+
+After that, rebuild libxkbcommon and xorg-server with xwayland support enabled. 
+
+Finally, rebuild all programs which required mesa and xorg-server. Use
+kiss-revdepends for a convenient list.
+
+Applications which use Qt and GTK will require you to specify the proper backend
+to use. GTK will default to running applications with Wayland. Qt applications
+will require you to specify xkb or Wayland for the backend, and supports a
+precedent-based list. Additionally, Qt applications will require qt5-wayland.
+
+See [16] for information on other backends and applications.
+
+Add the following to your env (eg $HOME/.profile)
+
++----------------+-------------------------------------------------------------+
+| GTK            | GDK_BACKEND=x11                                             |
+| Qt             | QT_QPA_PLATFORM=xcb,wayland                                 |
++----------------+-------------------------------------------------------------+
+
+
+[2.0] Xorgless
+________________________________________________________________________________
+
+Ensure no Xorg packages are installed, and remove any Xorg repositories from
+$KISS_PATH, keeping only xkeyboard-config and libxkbcommon. You will have to
+modify the depends files of many packages (including mesa, freetype-harfbuzz,
+pango, cairo, etc.) to not include any Xorg dependencies, but few modifications
+are required beyond that.
+
+First, build wayland and wayland-protocols.
+
+Then, build xkeyboard-config, followed by libxkbcommon with wayland support and
+without x11 support.
+
+Finally, build mesa with the wayland platform and without the x11 platform.
+
+Some packages might assume that, because you are building Linux, you must be
+using GLX. GLX is a graphics implementation provided by mesa, but is only built
+on Xorg systems. Wayland uses EGL and GLES. For some packages, like cairo, you
+might have to specify the gl-backend to be used. 
+
+Launching applications in Wayland requires a little more intervention. GTK will
+default to Wayland usually, but adding the following to your environment is a
+good idea
+
++----------------+-------------------------------------------------------------+
+| GTK            | GDK_BACKEND=wayland                                         |
+| Qt             | QT_QPA_PLATFORM=wayland                                     |
+| firefox        | MOZ_ENABLE_WAYLAND=wayland                                  |
++----------------+-------------------------------------------------------------+
+
+
+    [2.1] Alternatives 
+    ____________________________________________________________________________
+
+    See [9] for a larger list!
```

Comment:
There is also this curated list. https://arewewaylandyet.com/ . 
I am unsure on how viable it is to suggest alternatives. Given the drastic development this whole wayland is, this list will likely become outdated fast too, and therefore hard to maintain.
I suggest just to reference this awesome lists instead and spread the goodness of eg foot on those :p

Reply:
That would be simpler. Probably better.